### DISCUSSION TITLE
How to skip RNNotifee from TVOS target

### DISCUSSION THREAD

**Discussion by Tring-Srini** [react-native-community/discussions-and-proposals#760] [Feedback]
I'm using notifee for push notifications in mobile not for tvos. But everytime the TVOS native modules search for the notifee dependency and its throw an error

"Notifee Native module not found"
<img width="1042" alt="Screenshot 2024-02-05 at 11 03 49 AM" src="https://github.com/react-native-community/discussions-and-proposals/assets/55490094/382cbf6b-c413-406c-9332-c0f087c3036d">

I need to skip the native modules while use_native_modules! execution for tvos target. If Is it possible to skip then share your thoughts.

**Comment by cortinico** [comment]
You should report this to the TvOS or the Notifee repo
